src/meta/src/hummock/hummock_manager.rs

```diff
@@ -75,9 +74,9 @@ pub struct HummockManager<S: MetaStore> {
 pub type HummockManagerRef<S> = Arc<HummockManager<S>>;
 
 struct Compaction {
-    // TODO: `compact_status` will be managed per `compaction_groups` soon
-    compact_status: CompactStatus,
     compact_task_assignment: BTreeMap<u64, CompactTaskAssignment>,
+    // TODO: may want fine-grained lock for each compaction group
```

Comment:
if we already have different compact status for different group here, why do we need to filter in get_compact_task?

Reply:
`compact_status` doesn't store the LSMT aka `levels` .
The `levels` passed into `get_compact_task` is a global one. We can filter it before passing it to `get_compact_task` of course.

Reply:
> `compact_status` doesn't store the LSMT aka `levels` .
> The `levels` passed into `get_compact_task` is a global one. We can filter it before passing it to `get_compact_task` of course.

sorry. I forgot that it has been refactored.